From 031cd8dd376ed6537afd06ca5aec5e67f5da0489 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Fri, 31 Aug 2012 16:14:49 +0200
Subject: [PATCH] patch: soname

Squashed commit of the following:

commit 2035ced803168210a70c4946814440764e5d0186
Author: Nils Philippsen <nils@redhat.com>
Date:   Fri Aug 31 16:13:51 2012 +0200

    don't use the same SONAME for backend libs and main lib
---
 ltmain.sh | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/ltmain.sh b/ltmain.sh
index f3eb4c8..17d1508 100755
--- a/ltmain.sh
+++ b/ltmain.sh
@@ -8101,25 +8101,6 @@ EOF
 	  dlname=$soname
 	fi
 
-	# Local change for sane-backends: internal name for every lib
-	# is "libsane" not "libsane-backendname". So linking to each
-	# backend is possible. Also the following test was moved to this
-	# location.
-	# If -module or -export-dynamic was specified, set the dlname
-	if test "$module" = yes || test "$export_dynamic" = yes; then
-	  # On all known operating systems, these are identical.
-	  dlname="$soname"
-	fi
-	case $host in
-	  *mingw*)
-	    ;;
-	  *aix*)
-	    ;;
-	  *)
-	    soname=`echo $soname | sed -e "s/libsane-[A-Za-z_0-9]*/libsane/g"`
-	esac
-	# End of local change
-
 	lib="$output_objdir/$realname"
 	linknames=
 	for link
-- 
1.7.11.4

